attack_nonwindows.go

```diff
@@ -0,0 +1,9 @@
+// +build !windows
+
+package main
+
+import "flag"
+
+func systemSpecificFlags(fs *flag.FlagSet, opts *attackOpts) {
+	fs.Var(&opts.resolvers, "resolvers", "Override system dns resolution (comma separated list)")
```

Comment:
"List of addresses (ip:port) to use for DNS resolution. Disables use of local system DNS."